agent/src/beerocks/monitor/monitor_stats.cpp

```diff
@@ -280,7 +280,7 @@ void monitor_stats::process()
     //send response
     if (!requests_list.empty()) {
         auto request = requests_list.front();
-        LOG(TRACE) << "Serving reqeust: mid = " << request.message_id << " MAC = " << request.mac;
+        // LOG(TRACE) << "Serving reqeust: mid = " << request.message_id << " MAC = " << request.mac;
```

Comment:
You are my hero ❤️ 
BTW, do you think we should add a reference to the issue number?

Reply:
There are lots of bugs in the code with open issues on then. I think that adding issue number is more appropriate for a w/a